src/bundle/Resources/public/scss/ui/modules/sub-items-list/_table.view.item.scss

```diff
@@ -89,32 +89,6 @@
     &__priority-value {
         max-width: 100%;
         padding: 0 calculateRem(10px);
-
-        &[readonly] {
-            cursor: pointer;
-        }
-
-        &[readonly]::-webkit-outer-spin-button,
```

Comment:
If anyone wonders where it was added: https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/214